### DISCUSSION TITLE
Inventory Grid Rotatable Items

### DISCUSSION THREAD

**Discussion by drakonbane** [peter-kish/gloot#146] [Ideas]
Would it be possible to implement rotation for grid based items?

**Comment by peter-kish** [comment]
Yeah, rotating items by 90 degrees for easier inventory management has been on my mind since the beginning, but I haven't got around implementing it yet :(
I opened an issue that should server as a reminder: https://github.com/peter-kish/gloot/issues/147

**Comment by peter-kish** [comment]
This is now implemented in [v2.4.1](https://github.com/peter-kish/gloot/releases/tag/v2.4.1).

Right-click items inside an `InventoryGrid` to rotate them inside the editor, or call `InventoryGrid.rotate_item` from gdscript.

Note that this doesn't work inside a regular `CtrlInventoryGrid`, and to enable it `InventoryGrid.rotate_item` needs to be called when the `inventory_item_context_activated` signal is triggered (i.e. when an item is right-clicked).

See the [`InventoryGrid`](https://github.com/peter-kish/gloot/blob/master/docs/inventory_grid.md) documentation for more details.
